src/components/web3/api.ts

```diff
@@ -51,9 +63,17 @@ export const web3Login = async (): Promise<string> => {
   return allAddresses[0];
 };
 
+export const web3LoginSol = async (): Promise<string> => {
+  const result = await window.braveSolana.connect();
```

Comment:
@kdenhartog - here is what I don't understand. our documentation says `window.braveSolana.connect` - why have such an object if we also have `window.solana` ???

@hadiamjad - until we have a definitive answer on this, please continue to use `window.braveSolana`, but be prepared to use `window.solana` as a fallback... cf., https://bravesoftware.slack.com/archives/CBP2ZKC0H/p1687430956921859

Reply:
Oh, we probably need to update our docs then. I added a comment in that slack message because I realized as soon as you asked that question (thank you for raising it!) I realized directly calling `window.solana` is actually no longer the BCP. The `window.braveSolana` object was added so we could support wallet-standard. We could even simplify this logic a bit by reusing Solana lab's [component library](https://github.com/wallet-standard/wallet-standard/tree/master/packages/react/core).